src/utils.mli

```diff
@@ -0,0 +1,262 @@
+open Import
+
+val with_output : label option -> binary:bool -> f:(out_channel -> 'a) -> 'a
+
+module Kind : sig
+  type t = Intf | Impl
+
+  val of_filename : string -> t option
+
+  val describe : t -> string
+
+  val equal : t -> t -> bool
+end
+
+module Intf_or_impl  : sig
+  type t =
+  | Intf of signature
+  | Impl of structure
+
+  val map : t ->
+    < arg_label : arg_label -> arg_label;
```

Comment:
Can we use `Ast_traverse.map` here to make the signature shorter?

Reply:
Yees, so much better! Thanks!